src/core/js/models/itemsQuestionModel.js

```diff
@@ -159,6 +159,11 @@ define([
             this.set('_isAtLeastOneCorrectSelection', false);
         },
 
+        reset: function(type, force) {
+            QuestionModel.prototype.reset.call(this, type, force);
```

Comment:
`.apply(this, arguments);` is a bit more future-safe as it forwards all supplied arguments without bias.